install/helioviewer/db.py

```diff
@@ -634,7 +654,7 @@ def create_datasource_property_table(cursor):
 (81, 'Observatory', 'TRACE', 'TRACE', 'Transition Region and Coronal Explorer', 1),
 (81, 'Measurement', '1700', '1700', 'TRACE 1700', 2),
 (82, 'Observatory', 'TRACE', 'TRACE', 'Transition Region and Coronal Explorer', 1),
-(82, 'Measurement', 'white-light', 'WL', 'TRACE white-light', 2),
+(82, 'Measurement', 'white-light', 'white-light', 'TRACE white-light', 2),
```

Comment:
Is this from another PR? Not Solar Orbiter related...

Reply:
It is for this PR.
It is only somewhat related because I am updating data ingestion to use `fitsName` instead of `name` and this change is needed to keep the current ingestion behavior the same for that TRACE data source.